app/lib/features/activities/widgets/space_activities_section/item_widgets/type_widgets/attachment.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:acter/features/activities/widgets/space_activities_section/item_widgets/activity_item_container_widgets.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:acter_notifify/model/push_styles.dart';
+import 'package:flutter/material.dart';
+
+class ActivityAttachmentItemWidget extends StatelessWidget {
+  final Activity activity;
+  const ActivityAttachmentItemWidget({super.key, required this.activity});
+
+  @override
+  Widget build(BuildContext context) {
+    final activityObject = activity.object();
+    final name = activity.name();
+    final subType = activity.subTypeStr();
+    final objectEmoji = activityObject?.emoji();
+    final objectTitle = activityObject?.title();
+
+    return ActivityUserCentricItemContainerWidget(
+      actionTitle: '${PushStyles.attachment.emoji} Added attachment on',
+      actionObjectInfo: '$objectEmoji $objectTitle',
```

Comment:
_within_ the app we have actual icons we use rather than emojis and I think we should use those here. Emojis are for notifications because we can't provide the proper icons there...